### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [atom-ide-community/atom#390] [Q&A]
I saw you forked the flight manual,
and host a GitHub pages site.

Also most of the documentation in
the atom repository seems to point
to it - well to the official page -
anyways, so:

why not move the little remaining
documentation into the flight manual?

Have the documentation button, for
example, point directly to the
flight manual repo instead.

**Comment by kaosine** [comment]
Actually I'm working on putting the documentation from that flight-manual into the website under my [ac-website](https://github.com/kaosine/ac-website/) repo. It hasn't made it live to the repo here since I need to fix things.

However I wouldn't mind some help fixing things like this because there's a lot of it(and it probably means a lot of refactoring):

`   Liquid Warning: Liquid syntax error (line 11): Unexpected character / in "{{/danger}}" in /home/runner/ac-website/_docs/behind-atom/sections/maintaining-your-packages.md
    Liquid Warning: Liquid syntax error (line 15): Unexpected character # in "{{#note}}" in /home/runner/ac-website/_docs/behind-atom/sections/maintaining-your-packages.md
    Liquid Warning: Liquid syntax error (line 19): Unexpected character / in "{{/note}}" in /home/runner/ac-website/_docs/behind-atom/sections/maintaining-your-packages.md
    Liquid Warning: Liquid syntax error (line 37): Unexpected character # in "{{#danger}}" in /home/runner/ac-website/_docs/behind-atom/sections/maintaining-your-packages.md`

(note that's just a small taste of what needs to be fixed in all the files because I didn't want to overwhelm and overflow too much)

I'm still trying to remotely tweak the homepage how it should look with the tiles and get the navigation throughout the site to work as well. But there's so much that kind of needs to be fixed here and there from this old stuff and I'm trying to also document as I go so others can jump in once this is pushed up)

**Comment by ghost** [comment]
I'm a tech writer and use Atom for writing, so would be happy to help with the documentation.
